pkg/image/definition.go

```diff
@@ -28,6 +28,13 @@ type OperatingSystem struct {
 	Users      []OperatingSystemUser `yaml:"users"`
 	Systemd    Systemd               `yaml:"systemd"`
 	Suma       Suma                  `yaml:"suma"`
+	Packages   Packages              `yaml:"packages"`
+}
+
+type Packages struct {
+	PKGList  []string `yaml:"pkgList"`
+	AddRepos []string `yaml:"additionalRepos"`
+	RegCode  string   `yaml:"regCode"`
```

Comment:
Yes, this seems to be an misstep from my side.. Potentially we can use the same encryption that is used for the user passwords and decrypt the `RegCode` in the Go code. That said, this would require adding decryption logic to the EIB which would further increase the complexity of the PR.

If this is not a deal breaker I would prefer doing this as an iteration after this PR. If this is not possible, I'll be happy to do it in this PR, sorry for missing this..

Reply:
I vote for raising an issue and addressing it in a follow up. The current PR is too big as it is.

Reply:
+1 to issue and follow up